src/jsx/dom/client.ts

```diff
@@ -0,0 +1,73 @@
+import { useState } from '../hooks'
```

Comment:
Add a [module doc](https://jsr.io/docs/writing-docs#module-documentation) at the top, like #2796.

Reply:
OK, I added it in 61e2f7e.